use_case_implementation/fridge_object_detection/.gitignore

```diff
@@ -0,0 +1,372 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Can we add text in documentation at top level how we used the repo, what changes were done for the implementation

Reply:
I have added this in the Readme inside the implementation folder. `Adaptation from the accelerator and differences` Please let me know whether that will do.